lib/modules/manager/composer/utils.ts

```diff
@@ -118,18 +123,6 @@ export function findGithubToken(
   return searchResult?.token?.replace('x-access-token:', '');
 }
 
-export function isGithubPersonalAccessToken(token: string): boolean {
-  return token.startsWith('ghp_');
-}
-
-export function isGithubServerToServerToken(token: string): boolean {
-  return token.startsWith('ghs_');
-}
-
-export function isGithubFineGrainedPersonalAccessToken(token: string): boolean {
-  return token.startsWith('github_pat_');
-}
-
 export function takePersonalAccessTokenIfPossible(
```

Comment:
I found there are more GitHub specific utils in `lib/modules/manager/composer/utils.ts`:

- `findGithubToken()`
- `takePersonalAccessTokenIfPossible()`

Should them be moved to `lib/util/check-token.ts` too?

Reply:
Please go ahead